rest/route/task_generate.go

```diff
@@ -47,6 +52,29 @@ func (h *generateHandler) Run(ctx context.Context) gimlet.Responder {
 	if err := data.GenerateTasks(h.taskID, h.files); err != nil {
 		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "generating tasks for task '%s'", h.taskID))
 	}
+	t, err := task.FindOneId(h.taskID)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting task '%s'", h.taskID))
+	}
+	if t == nil {
+		return gimlet.MakeJSONErrorResponder(errors.Errorf("task '%s' not found", h.taskID))
+	}
+	ts := utility.RoundPartOfMinute(1).Format(units.TSFormat)
+	j := units.NewGenerateTasksJob(t.Version, t.Id, ts)
+	queueName := fmt.Sprintf("service.generate.tasks.version.%s", t.Version)
+	queue, err := evergreen.GetEnvironment().RemoteQueueGroup().Get(ctx, queueName)
+	if err != nil {
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "getting generate tasks queue '%s' for version '%s'", queueName, t.Version))
+	}
+	grip.Info(message.Fields{
```

Comment:
Do we need this log?

Reply:
I think it's nice to have but should probably move it to after we successfully enqueue, that being said if you think it's too noisy I can remove.

Reply:
Eh, I don't have strong feelings, it just seems like it's a duplicate since we could trace that it succeeded based on the REST request started/completed logs that already go to Splunk (and the presence/absence of Amboy job logs).